src/containers/detail/Detail.tsx

```diff
@@ -38,47 +40,17 @@ export class Detail extends Component<RouteComponentProps<DetailProps>, DetailSt
-				lom_classification: [
-					'beeldende kunsten - schilderkunst, beeldhouwkunst, bouwkunst',
-					'fotografie - video - film',
-					'gedrags- en cultuurwetenschappen',
-					'muzikale opvoeding - muziek',
-					'PAV - MAVO - GASV - ASPV',
-				],
-				lom_typical_age_range: ['Secundair 2de graad', 'Secundair 3de graad'],
-				lom_intended_enduser_role: ['Docent', 'Student'],
-				duration_time: '00:08:32',
-				duration_seconds: 512,
-				administrative_type: 'video',
-				administrative_external_id: '8911n96442',
-			},
+			item: {},
 		};
 		// TODO: get item from store by id
-		// const itemId: string = (props.match.params as any)['id'];
+		const itemId: string = (props.match.params as any)['id'];
```

Comment:
Installeer de @types van react-router(-dom) hier om correct te kunnen typen

Reply:
die types zijn reeds geïnstalleerd. de cast naar any gebeurd omdat ik een router param ophaal en de types in react-router-dom het object `props.match.params` getyped hebben als `{}`. dus id bestaat niet op {}.
Je kan blijkbaar wel een interface schrijven per router param, maar dat lijkt me wat overkill: https://stackoverflow.com/a/46899010/373207